api/db/changelog/db.changelog-9-userworkspace.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="brubenst" id="changelog-9-userworkspace">
+    <createTable tableName="user_workspace">
+      <column name="user_workspace_id" type="bigint" autoIncrement="true">
```

Comment:
Did you look into using a compound primary key on user id, workspace id instead?